ASU2019_NetworkedGameWorkshop/controller/GameManager.cs

```diff
@@ -206,21 +207,21 @@ private enum GameStage { Buy, Fight, FightToBuy, BuyToFight }
                 }
                 return true;
             }
-            foreach(Character character in TeamBlue) {
+            foreach(Character character in TeamBlue.Where(e => !e.IsDead)) {
                 updateCanvas = character.update() || updateCanvas;
             }
             TeamBlue = TeamBlue.Where(predicate).ToList();

```

Comment:
you don't draw dead characters and u don't loop over them in the foreach then why even remove them ?